### ISSUE TITLE
Is it also possible to record the chat in any way?

### ISSUE THREAD

**Issue by CodeChrist** [Michele0303/tiktok-live-recorder#69]
Thanks for the great work, works nicely, just wanted to know if its somehow possible to also capture the chat?

Thanks and best regards

**Comment by Michele0303** [comment]
Yes, it would be possible to record the chat and save it in a txt file

**Comment by CodeChrist** [comment]
> Yes, it would be possible to record the chat and save it in a txt file

Thanks alot for the reply, I guess having it viewable inside the video file is propably alot more difficult?

**Comment by Michele0303** [comment]
Yes

**Comment by CyberbobGR** [comment]
> Yes, it would be possible to record the chat and save it in a txt file

Please add that feature, it would be awesome! 😍

**Comment by takimakura** [comment]
Would showing gift effects for the ones not showing up at the moment be difficult?

**Comment by vioan** [comment]
this will be a great addition, thanks!

**Comment by eliich** [comment]
This other package has the ability to record live mp4, comments, and gifts. https://github.com/isaackogan/TikTokLive
